invenio_communities/templates/semantic-ui/invenio_notifications/community-invitation.submit.jinja

```diff
@@ -0,0 +1,56 @@
+{% set invitation_request = notification.context.request %}
+{% set receiver = invitation_request.receiver %}
+{% set community = invitation_request.created_by %}
+{% set request_id = invitation_request.id %}
+
+{% set community_title = community.metadata.title %}
+{% set message = notification.context.message | safe %}
+{% set role = notification.context.role %}
+
+{# TODO: use request.links.self_html when issue issue is resolved: https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 #}
+{% set request_link = "{ui}/me/requests/{id}".format(
+    ui=config.SITE_UI_URL, id=request_id
+    )
+%}
+
+{%- block subject -%}
+New invitation to join community {{ community_title }} as {{ role }}
```

Comment:
shouldn't these strings be translated here instead of https://github.com/inveniosoftware/invenio-communities/pull/967/files#diff-005172a68d8456c6b161cd565873088a9a3a574d928ffde1ba3d7d94cf1e0345R281 ? I am a bit lost on why we have description also set in service